rbac/management/group/definer.py

```diff
@@ -31,7 +31,8 @@ def seed_group(tenant):
     """For a tenant create or update default group."""
     with tenant_context(tenant):
         name = 'Default user access'
-        group_description = 'Default group that contains default permissions for new users.'
+        group_description = 'These are the roles all users inherit by default.\
```

Comment:
Thanks for refining the description, now it looks much better! 
How about change "These are the roles all ..." to "This group defines the roles all ..." ?